Add missing `Shift` to Bold editor ordered & unordered list keyboard shortcuts

etmoore opened the pull request:
Noticed shift key was missing from ordered and unordered list commands. See redactor docs: https://imperavi.com/redactor/docs/get-started/shortcuts/#s-predefined%20shortcuts

mobitar commented:
I think while we're at it, we can make this just a little more clear to save the person reading some researching: you can use the desktop app for testing your extensions to avoid mixed content issues, or use a web app hosted on localhost.

MaxLap commented:
I agree. I improved the text and its location. You can take another look now

mobitar commented on app/views/website/markdown/extensions/building-an-extension.md:
```diff
@@ -35,10 +35,12 @@ In this example, we'll use our blank-slate ReactJS template to build a utility b
 1. In your browser, open `http://localhost:8080/ext.json`, and ensure that the output matches the file contents of the ext.json file. Most importantly, if your dev server is running on a different port, make sure to update the `url` property of the JSON file to reflect the correct value.
 
-1. In **Standard Notes**, click **Extensions** in the lower left corner of the app, click **Import Extension** in the bottom right of the **Extensions** window, and input your ext.json location: `http://localhost:8080/ext.json`. Then press enter.
+1. In the actual **Standard Notes** application (not via a browser), click **Extensions** in the lower left corner of the app, click **Import Extension** in the bottom right of the **Extensions** window, and input your ext.json location: `http://localhost:8080/ext.json`. Then press enter.
```
The web app is no less actual than our desktop app ;)

How about:

> Using the Standard Notes **desktop application**, ...

MaxLap replied:
Haha, you are right.

Changed to: the **Standard Notes** desktop application (browser may not work for this)

mobitar replied:
👌

etmoore committed:
Add `shift` to list-related keyboard shortcuts

Noticed shift key was missing from ordered and unordered list commands. See redactor docs: https://imperavi.com/redactor/docs/get-started/shortcuts/#s-predefined%20shortcuts

etmoore commented:
As an aside, I've also noticed that (in the Bold editor) if you type `<header level>.<space>`, it'll apply the header formatting.
eg. 
1. on a new line, type "h1."
2. hit space
3. 🎉  the Heading 1 format is now applied!

This is cool + handy, but I can't find documentation for it anywhere. Was hoping there was a similar `code.` shortcut. But alas…

theodorechu commented:
Thanks for catching this!
